index.html

```diff
@@ -0,0 +1,203 @@
+                        your team's money and digital assets.</p>
+                </div>
+                <div class="schedule">
+                    <h2>2015.10.15(THU) ~ 16(FRI)</h2>
+                    <h4>@ National Museum of Bangladesh, Art Center and more
+                    </h4>
+                </div>
+            </div>
+        </section>
+    </section>
+    <!-- main program section -->
+
+    <main>
+        <div class="main-head">
+            <h3>Main Program</h3>
+            <hr>
+        </div>
+        <div class="main-cards">
+            <div class="main-card">
+                <img src="./images/icon-1.png" alt="icon">
+                <h4>Wires & ACH</h4>
+                <h6>Same/next day transfer to bank accounts</h6>
+            </div>
+            <div class="main-card">
+                <img src="./images/icon-2.png" alt="icon">
+                <h4>Checks</h4>
+                <h6>Send and receive physical checks</h6>
+            </div>
+            <div class="main-card">
+                <img src="./images/icon-3.png" alt="icon">
+                <h4>Cards</h4>
+                <h6>Spend your funds in the real world</h6>
+            </div>
+            <div class="main-card">
+                <img src="./images/icon-4.png" alt="icon">
+                <h4>On-chain</h4>
+                <h6>
+                    Move your funds across chains in a few clicks</h6>
+            </div>
+            <div class="main-card">
+                <img src="./images/icon-5.png" alt="icon">
+                <h4>Deposite</h4>
+                <h6>Spend your funds in the real world</h6>
+            </div>
+        </div>
+        <div class="main-btn">
+            <button>Join CC Global Summit 2015</button>
+            <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+    </main>
+
+    <!-- featured speakers -->
+
+    <section class="speaker" id="speaker">
+        <div class="speaker-head">
+            <h3>Featured Speakers</h3>
+            <hr>
+        </div>
+        <div class="speaker-cards" id="speaker-cards">
+            <!-- load data from js file -->
+
+            <div class="speaker-btn" id="load-more-btn">
+                <button>MORE <img src="./images/arrow-down.png" alt="arrow-down icon"></button>
+            </div>
+        </div>
+    </section>
+
+    <!--sponsors section -->
+
+    <section class="sponsors">
+        <div class="sponsors-body">
+            <div class="sponsors-head">
+                <h3>Partner</h3>
+                <hr>
+            </div>
+            <div class="sponsors-brand">
+                <img src="./images/mozilla 1.png" alt="mozilla">
+                <img src="./images/googlekorea 1.png" alt="googlekorea">
+                <img src="./images/naver 1.png" alt="naver">
+                <img src="./images/daumkakaologo 1.png" alt="daumkakaologo">
+                <img src="./images/airbnb 1.png" alt="airbnb">
+            </div>
```

Comment:
- [ ] Please kindly reduce the size of this images so that we can have a design that is close to the given template. 😄